services/cards-consultation/src/main/java/org/opfab/cards/consultation/model/LightCard.java

```diff
@@ -65,6 +67,7 @@ public class LightCard {
     private String wktProjection;
     private Integer secondsBeforeTimeSpanForReminder;
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @JsonProperty("rRule")
```

Comment:
As it is tricky , to avoid removing it in the future , add comment : `// if we don't use this annotation, the field will be serialized as "rrule"`